Core/include/Acts/Propagator/Propagator.ipp

```diff
@@ -256,6 +284,36 @@ auto Acts::Propagator<S, N>::makeResult(propagator_state_t state,
   moveStateToResult(state, result);
 
   if (makeCurvilinear) {
+    if constexpr (has_extension<decltype(state.stepping)>::value &&
```

Comment:
Not clear whether this is sufficient to test whether the enclosed lines are valid